src/qp/metrics/pit.py

```diff
@@ -0,0 +1,140 @@
+
+    @property
+    def pit_samps(self):
+        """Return the samples used to compute the PIT"""
+        return self._pit_samps
+
+    @property
+    def pit(self):
+        """Return the PIT Ensemble object"""
+        return self._pit
+
+    def calculate_pit_meta_metrics(self):
+        """Convenience method that will calculate all of the PIT meta metrics and return them 
+            as a dictionary.
+
+        Returns:
+            dict: The collection of PIT statistics
+        """
+        pit_meta_metrics = {}
+
+        pit_meta_metrics['ad'] = self.evaluate_PIT_anderson_ksamp()
+        pit_meta_metrics['cvm'] = self.evaluate_PIT_CvM()
+        pit_meta_metrics['ks'] = self.evaluate_PIT_KS()
+        pit_meta_metrics['outlier_rate'] = self.evaluate_PIT_outlier_rate()
+
+        return pit_meta_metrics
+
+    def evaluate_PIT_anderson_ksamp(self, pit_min=0., pit_max=1.):
+        """Use scipy.stats.anderson_ksamp to compute the Anderson-Darling statistic
+            for the cdf(truth) values by comparing with a uniform distribution between 0 and 1.
+            Up to the current version (1.9.3), scipy.stats.anderson does not support
+            uniform distributions as reference for 1-sample test, therefore we create a uniform
+            "distribution" and pass it as the second value in the list of parameters to the scipy 
+            implementation of k-sample Anderson-Darling. 
+            For details see: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.anderson_ksamp.html
+
+        Args:
+            pit_min (float, optional): Minimum PIT value to accept. Defaults to 0..
+            pit_max (float, optional): Maximum PIT value to accept. Defaults to 1..
+
+        Returns:
+        output [Objects]: A array of objects with attributes `statistic`, `critical_values`, and `significance_level`.
+        For details see: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.anderson_ksamp.html
+        """
+        # Removed the CDF values that are outside the min/max range
+        trimmed_pit_values = self._trim_pit_values(pit_min, pit_max)
+
+        uniform_yvals = np.linspace(pit_min, pit_max, len(trimmed_pit_values))
+
+        return quick_anderson_ksamp(trimmed_pit_values, uniform_yvals)
+
+    def evaluate_PIT_CvM(self):
+        """Calculate the Cramer von Mises statistic using scipy.stats.cramervonmises using self._pit_samps
+        compared to a uniform distribution. For more details see:
+        https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.cramervonmises.html
+
+        Returns:
+            output [Objects]: A array of objects with attributes `statistic` and `pvalue`
+            For details see: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.cramervonmises.html
+        """
+        return stats.cramervonmises(self._pit_samps, 'uniform')
+
+    def evaluate_PIT_KS(self):
+        """_summary_
```

Comment:
\_comment\_

Reply:
Oops, I clearly missed this one :) I'll fix this.

Reply:
I updated the docstring here.